src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj

```diff
@@ -11,7 +11,8 @@
     <NoStdLib>true</NoStdLib>
     <ApplyNgenOptimization Condition="'$(TargetFramework)' == 'netstandard2.0'">partial</ApplyNgenOptimization>
     <GeneratePerformanceSensitiveAttribute>true</GeneratePerformanceSensitiveAttribute>
-    
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
This is a temporary change correct? The idea being that when we are done with this effort that we will be adding this to `Directory.Build.props`? If so are we tracking that in an issue, comment, etc ...?

Reply:
Filed #51099 to track. There is a bug in wpf temp projects that prevent us from enabling this in Settings.props or Directory.Build.props for now (is one or the other preferred? Directory.Build.props seemed pretty empty.)

I should probably file the bug to get this property fixed with WPF projects. Do you know the right repo to do that in? Maybe @clairernovotny knows? It seems not so bad to exclude those projects from rebuilding for a while though.

Reply:
The WPF issues with targets/props and source link should all be fixed by setting `IncludePackageReferencesDuringMarkupCompilation` to true in the `Directory.Build.Props` file and using a 5.0.200+ SDK to build. This enables fixes within the WPF targets that are currently opt-out.